data/demo/config.edn

```diff
@@ -20,5 +20,5 @@
                      :output-dir "results"}
 
  :validation-parameters {:keep-temp-files? false
-                         :run-as-default true}
+                         :run-as-default false}
```

Comment:
It was annoying me that every time I ran `-main` when working on the model, the validation was running as well.